### ISSUE TITLE
Problem if vcap file contains both credentials IAM and username/password

### ISSUE THREAD

**Issue by markheger** [IBMStreams/streamsx.topology#1494]
Using TopologyTester (1.8.5) and Streams RH6.x cloud vm with new plans.

The Streaming Analytics plan "**Enhanced VM Hourly**" contains IAM parameters and username and password.
Response returned with error code: 401
{"trace":"4178f99e6e534475bb770e83e68f924d","errors":[{"code":"unauthorized","message":"UNAUTHORIZED CDISB4010E Requested file or operation is not accessible with current authorization."}]}

If username and password is removed from vcap file, then the following error occurs:
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://streams-console-c6d1.ng.bluemix.net/streams/rest/resources

**Comment by ddebrunner** [comment]
It's failing on the cancel job request against the service.

So it successfully connects to the service REST api to submit the job, and the Streams REST api to monitor the test, but fail on the service REST api cancel.

**Comment by ddebrunner** [comment]
The failed cancel:
```
requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://streams-app-service.ng.bluemix.net/v2/streaming_analytics/a59dc5af-9e46-4c0b-92ac-b45e692d46bf/jobs/5
```

**Comment by ddebrunner** [comment]
Fixed by #1496

**Comment by markheger** [comment]
1.8.6 works fine. Thanks for quick fix.
